pyth_observer/check/publisher.py

```diff
@@ -134,11 +138,25 @@ def __init__(self, state: PublisherState, config: PublisherCheckConfig):
         self.__state = state
         self.__max_slot_distance: int = int(config["max_slot_distance"])
         self.__abandoned_slot_distance: int = int(config["abandoned_slot_distance"])
+        self.__market_open: bool = is_market_open(
+            self.__state.asset_type.lower(),
+            datetime.now(ZoneInfo("America/New_York")),
+        )
 
     def state(self) -> PublisherState:
         return self.__state
 
+    def abandoned_slot_distance(self) -> int:
+        return self.__abandoned_slot_distance
+
+    def market_open(self) -> bool:
+        return self.__market_open
```

Comment:
added these functions because we need to access these states in `check_publisher()`